admin/client/views/reviewList.js

```diff
@@ -0,0 +1,399 @@
+'use strict';
+
+import React from 'react';
+import update from 'react/lib/update';
+import xhr from 'xhr';
+import async from 'async';
+import CurrentListStore from '../stores/CurrentListStore';
+import CreateReviewModal from '../components/CreateReviewModal';
+import ReviewDndContainer from './reviewDndContainer';
+import FlashMessages from '../components/FlashMessages';
+import Footer from '../components/Footer';
+import MobileNavigation from '../components/MobileNavigation';
+import PrimaryNavigation from '../components/PrimaryNavigation';
+import SecondaryNavigation from '../components/SecondaryNavigation';
+import { BlankState, Button, Container, InputGroup, ResponsiveText, Spinner } from 'elemental';
+import { plural } from '../utils';
+
+const latestSavePanelStyle = {
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'flex-end',
+  marginBottom: '32px',
+};
+
+const latestColumnContainerStyle = {
+  borderBottomWidth: '2px',
+  borderBottomStyle: 'solid',
+  borderBottomColor: 'rgba(0, 0, 0, 0.06)',
+  paddingBottom: '10px',
+  marginBottom: '10px',
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  paddingLeft: '68px',
+};
+
+const columnTextStyle = {
+  color: '#999',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
```

Comment:
ref: [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow)

Reply:
style property with two names `ex. text-overflow` needs to be passed by camelcase in react ([ref](https://www.w3schools.com/react/react_css.asp))

Reply:
![image](https://github.com/twreporter/keystone/assets/61230075/d7de43c0-32ec-46c1-8300-a3e81f04a652)
need to add `white-space: nowrap`